vc_zoom/setup.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import unicode_literals
+
+from setuptools import find_packages, setup
+
+
+setup(
+    name='indico-plugin-vc-zoom',
+    version='0.3-dev',
+    description='Zoom video-conferencing plugin for Indico',
+    url='',
```

Comment:
link to the github repo (probably best to sync the setup.py content with another plugin)